src/containers/MathObjects/components/StatusSymbol/components/Swatch.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropType from 'prop-types'
+import styled from 'styled-components'
+import ColorSquare from './ColorSquare'
+
+const Container = styled.div`
+  display: grid;
+  grid-template-columns: repeat(5, 30px);
+  grid-auto-rows: 30px;
+  grid-gap: 5px;
+`
+
+export default class Swatch extends React.Component {
```

Comment:
@stardust66 I should have commented about this before, but I don't think Swatch is a good name for this component. According to Google, 

> > **Swatch:** a small sample of fabric intended to demonstrate the look of a larger piece.

In a sense, your ColorSquare are "swatches". 

"ColorPicker" Might be a reasonable name. Or "ColorPalette" since the number of choices are finite.